site/docs/javadoc/0.9.0/serialized-form.html

```diff
@@ -1031,7 +1031,7 @@ <h4>spec</h4>
 <li class="blockList"><a id="org.apache.iceberg.spark.source.SparkBatchWrite.TaskCommit">
 <!--   -->
 </a>
-<h3>Class org.apache.iceberg.spark.source.SparkBatchWrite.TaskCommit extends org.apache.iceberg.spark.source.TaskResult implements Serializable</h3>
+<h3>Class org.apache.iceberg.spark.source.SparkBatchWrite.TaskCommit implements Serializable</h3>
```

Comment:
The Javadoc for a release should not be modified. I think this is probably a search and replace error.

Reply:
You are right, we should not change the javadoc of 0.9.0 release.